### ISSUE TITLE
Creating an AdoptOpenJDK Specific Mailing List

### ISSUE THREAD

**Issue by SeanC1083** [AdoptOpenJDK/TSC#43]
This Git Issue was created following a conversation with Martijn (@karianna) about both the lack of a dedicated, unique Mailing List for AdoptOpenJDK, and the goal of improving the accessibility and the need for long-term content that users want to see.

**Comment by karianna** [comment]
We have adoption-discuss@openjdk.java.net - but it's not really appropriate for this community anymore

**Comment by karianna** [comment]
See #32 as well

**Comment by SeanC1083** [comment]
What kind of information do we expect to include in the mailing list and what users are we expecting to go there? That's an open question for anyone.  It's my understanding that Mailing List is one of the preferred ways for users to get and receive product updates, and other software release info.

**Comment by smlambert** [comment]
As with https://github.com/AdoptOpenJDK/TSC/issues/32, the move to Eclipse Foundation means there are several Adoptium project mailing lists, including the [pmc mailing list](https://projects.eclipse.org/projects/adoptium/contact), so will close this issue.
